src/views/Game/Game.styl

```diff
@@ -60,7 +60,7 @@ goban-view-bar-width=400px
 .MainGobanView {
     display: flex;
     flex-direction: row;
-    align-items: stretch;
+    align-items: stretch; // this is the default, I wonder why we need to say it?
```

Comment:
Answer (1) - if it's not needed, lets remove it :)

Answer (2) - technically it's not the default `normal` is the default which apparently behaves like `stretch` for grid items or `start` for grid items with a defined block size.  I don't know if this distinction is actually applicable in this case, but it might be?

Regardless, feel free to remove if everything is working fine without it.